collect_app/src/main/java/org/odk/collect/android/application/initialization/ExistingProjectMigrator.kt

```diff
@@ -56,7 +56,11 @@ class ExistingProjectMigrator(
             }
         }
 
-        FileUtils.deleteDirectory(File(rootDir, ".cache"))
+        try {
+            FileUtils.deleteDirectory(File(rootDir, ".cache"))
```

Comment:
I think it would be good to have a test to ensure this remains in place. The test (which can go in `ExistingProjectMigrator`) could just delete the cache directory before running `run()` and just check that nothing explodes - just a "does not explode test" but still important.

Reply:
Ok I added a test but making it a normal file not dir because only then I can force it to throw the exception. If I delete a dir that does not exist everything is fine because `deleteDirectory`() is our utility method and it checks that case. Hmm now I think maybe we should improve that method too/instead?